app/views/admin/users/show.html.erb

```diff
@@ -9,3 +9,31 @@
   <li><b>Small business</b>: <%= @view_model.small_business %></li>
   <li><b>Contracting officer</b>: <%= @view_model.contracting_officer %></li>
 </ul>
+
+<% if @view_model.bids? %>
+<h2>Bidding History</h2>
+
+<table class="usa-table-borderless" id="bidding-history">
+  <thead>
+    <tr>
+      <th scope="col">Auction</th>
+      <th scope="col">Start Date</th>
+      <th scope="col">End Date</th>
+      <th scope="col">User Bids</th>
+      <th scope="col">User Won?</th>
+      <th scope="col">Accepted?</th>
```

Comment:
Looks like these labels are slightly different from the ones in the original issue -- cool with me if you think these are clearer, just wanted to point that out in case it was an oversight